syntax = "proto3";

package LAMARCK_ML;

import "LAMARCK_ML/data_util/Attribute.proto";


message DatasetProto {
    string name_val = 1;
    string cls_name = 2;

    repeated AttributeProto attr_val = 11;
}